Invalid specifier in 'python_requires' of setup.py

Getting an error when running the following command in my local copy of the Blatann project:

```
$ make binaries
error in blatann setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.7.*'
make: *** [Makefile:39: binaries] Error 1
```

Perhaps my environment is more up-to-date with `setuptools` at version 68.2.2

It appears to work when I change the specifier to `>=3.7.0` in the `setup.py` - so that might hint at an update in the tools for packaging that might just show in my "fresh" development environment.

--- a/setup.py
+++ b/setup.py
@@ -24,7 +24,7 @@ setup(
     keywords="ble bluetooth nrf52 nordic",
     packages=find_packages(exclude=["tests", "tests.*"]),
     install_requires=install_requires,
-    python_requires=">=3.7.*",
+    python_requires=">=3.7",
     long_description_content_type="text/markdown",
     long_description=long_description,
     classifiers=[

